runtime/syntax/man.vim

```diff
@@ -5,18 +5,21 @@ if exists('b:current_syntax')
   finish
 endif
 
-syntax case  ignore
-syntax match manReference      display '[^()[:space:]]\+([0-9nx][a-z]*)'
-syntax match manSectionHeading display '^\S.*$'
-syntax match manTitle          display '^\%1l.*$'
-syntax match manSubHeading     display '^ \{3\}\S.*$'
-syntax match manOptionDesc     display '^\s\+\%(+\|-\)\S\+'
+" Not contained because sometimes, manpages contain garbage backspaced
+" characters. See Restricted Shell at bash(1)
```

Comment:
I guess since this kind of physical printing isn't common any more, it just looks like an inefficient binary instruction.  You might be too young and missed out on watching a dot matrix printer buzzing through its tape on the top shelf of a desk [such as this one][1].  Your heart raced every time your desk shook as the printer fed a new line, loosening, ever so slightly, the screws that held it all together.  It was a rush knowing you were risking your life printing homework because you couldn't find the allen wrench for this out-of-place piece of furniture that was holding a 30lb printer up above your head.  But, oh man was it worth it!  Once it was all done, you tore the curtain of paper away from the printer and gently peeled the trim off the sides.  You were left with a great sense of accomplishment and a handful of trim that your sister used to make bracelets.  You told your sister that her bracelets sucked, but you were secretly jealous of her creativity.  So many emotions swirled around this magic machine.

Anyways, what was I talking about?  Oh yeah, this is called `overstrike` printing.  Dot matrix printers didn't do this.  It was impact/hammer printers like the [daisy wheel][2] or a standard typewriter that used this method.  The print position is backed up by one character and stamped over.  To black out mistakes on a typewriter, X's or #'s were typically used.

I still can't find the reference, but you shouldn't really need it to interpret what you're seeing.  Off the top of my head, these could be created by using characters available on a daisy wheel:

`a^H~` = `ã`
``e^H` `` = `è`
`Y^H=` = `¥`
`C^H=` = `€`
`/^H=` = `≠`
`O^H|` = `ϕ`
`!^H?` = `‽`

Those are just examples to drive the point home for posterity.  I doubt any of those would be found in man pages.

[1]: https://share.esdf.io/HC35Ydk0jY/Songmics-Computer-Desk-Study-Table-Portable-Trolley-Home-_1.jpg
[2]: https://en.wikipedia.org/wiki/Daisy_wheel_printing